gradle.properties

```diff
@@ -13,6 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-version=4.1.0
+version=4.1.1_RC1
```

Comment:
Will you update this later?

Reply:
Yes, was planning on changing after merging.